theseus/extlib/tests/test_baspacho.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+import pytest  # noqa: F401
+import torch  # needed for import of Torch C++ extensions to work
+from scipy.sparse import csr_matrix, tril
+
+from theseus.utils import random_sparse_binary_matrix
+
+
+# ideally we would like to support batch_size <= init_batch_size, but
```

Comment:
Is this also true for Baspacho or is it remains of a copy paste from LUCuda?

Reply:
Oh yeah good catch. I thought this applied here.

Reply:
Remove comment then?

Reply:
cleared.